composer.json

```diff
@@ -85,6 +85,7 @@
         "symfony/yaml": "self.version"
     },
     "require-dev": {
+        "ext-zip": "*",
```

Comment:
This should be reverted as it does not seem to be related to the fix.

Reply:
@fabpot Should I create a separate PR for this ?

Reply:
I don't think this change is valid at all. IIRC we need the extension for some parts.

Reply:
for which part ? AFAIK, we don't need ext-zip to run tests (and then, we should mark the relevant tests as skipped instead of forbidding to install the dev environment).
And if this is about running composer, it does not require ext-zip. It requires having *either* ext-zip or the `unzip` command (i.e. having *something* to unzip). And it would still not be a requirement of Symfony.

Reply:
Our `simple-phpunit` script uses the zip extension to extract the downloaded PHPUnit archive.

Reply:
The problem I had was that I cloned the repository and ran the tests directly without having the zip extension installed. This gave me some errors. After install the php-zip extension everything worked fine. That's why I also proposed this change. But as it's not related to the ticket this PR is linked, I'll revert this.